crates/sp-domains/src/fraud_proof.rs

```diff
@@ -345,3 +349,64 @@ pub struct ImproperTransactionSortitionProof {
     /// Hash of the bad receipt this fraud proof targeted
     pub bad_receipt_hash: ReceiptHash,
 }
+
+#[derive(Clone, Debug, Decode, Encode, Eq, PartialEq, TypeInfo)]
+pub struct TotalRewardsProof<BlockNumber, BlockHash> {
+    /// The id of the domain this fraud proof targeted
+    pub domain_id: DomainId,
+    /// Hash of the bad receipt this fraud proof targeted
+    pub bad_receipt_hash: ReceiptHash,
+    /// Number of the block
+    pub block_number: BlockNumber,
+    /// Hash of the domain block corresponding to `block_number`.
+    pub domain_block_hash: BlockHash,
+    /// Storage witness needed for verifying this proof.
+    pub storage_proof: StorageProof,
+}
+
+/// Represents an invalid bundle fraud proof.
+#[derive(Clone, Debug, Decode, Encode, Eq, PartialEq, TypeInfo)]
+pub enum InvalidBundleProof<BlockNumber, BlockHash> {
+    TotalRewards(TotalRewardsProof<BlockNumber, BlockHash>),
+}
+
+impl<BlockNumber, BlockHash> InvalidBundleProof<BlockNumber, BlockHash> {
+    pub(crate) fn domain_id(&self) -> DomainId {
+        match self {
+            InvalidBundleProof::TotalRewards(proof) => proof.domain_id,
+        }
+    }
+
+    pub(crate) fn bad_receipt_hash(&self) -> ReceiptHash {
+        match self {
+            InvalidBundleProof::TotalRewards(proof) => proof.bad_receipt_hash,
+        }
+    }
+}
+
+/// This is a representation of actual Block Rewards storage in pallet-operator-rewards.
+/// Any change in key or value there should be changed here accordingly.
+pub struct OperatorBlockRewards;
```

Comment:
I found an equivalent of this: `frame_support::storage::storage_prefix("OperatorRewards".as_ref(), "BlockRewards".as_ref())`